habitat-baselines/habitat_baselines/config/experiments_hab3/socialnav_human_robot_oracle.yaml

```diff
@@ -0,0 +1,194 @@
+# @package _global_
+
+defaults:
+  - /benchmark/rearrange: multi_agent_tidy_house
+  - /habitat_baselines: habitat_baselines_rl_config_base
+  - /habitat/simulator/sim_sensors@habitat_baselines.eval.extra_sim_sensors.third_rgb_sensor: third_rgb_sensor
+  - /habitat_baselines/rl/agent: pop_play
+  - /habitat_baselines/rl/policy/obs_transforms@habitat_baselines.rl.policy.agent_0.obs_transforms.add_virtual_keys:
+    - add_virtual_keys_base
+  - /habitat_baselines/rl/policy/obs_transforms@habitat_baselines.rl.policy.agent_1.obs_transforms.add_virtual_keys:
+    - add_virtual_keys_base
+  - /habitat_baselines/rl/policy@habitat_baselines.rl.policy.agent_0: hl_fixed # monolithic
+  - /habitat_baselines/rl/policy@habitat_baselines.rl.policy.agent_1: hl_fixed #hl_fixed #hab3_hl_neural
+  - /habitat_baselines/rl/policy/hierarchical_policy/defined_skills@habitat_baselines.rl.policy.agent_0.hierarchical_policy.defined_skills: oracle_skills_ma
+  - /habitat_baselines/rl/policy/hierarchical_policy/defined_skills@habitat_baselines.rl.policy.agent_1.hierarchical_policy.defined_skills: oracle_skills_ma_humanoid
+  - /habitat/task/measurements:
+    - social_nav_reward
+  - /habitat/task/lab_sensors:
+    - relative_resting_pos_sensor
+    - target_start_sensor
+    - goal_sensor
+    - joint_sensor
+    - is_holding_sensor
+    - end_effector_sensor
+    - target_start_gps_compass_sensor
+    - target_goal_gps_compass_sensor
+    - localization_sensor
+    - humanoid_joint_sensor
+  - /habitat/task/actions@habitat.task.actions.agent_0_arm_action: arm_action
+  - /habitat/task/actions@actions.agent_0_oracle_nav_action: oracle_nav_action
+  - /habitat/task/actions@actions.agent_1_oracle_nav_action: oracle_nav_action
+  - override /habitat/task/rearrange: multi_agent_tidy_house_base
+  # For Spot observation space
+  - override /habitat/simulator/agents@habitat.simulator.agents.agent_0: rgbd_head_rgbd_arm_agent
+  - _self_
+
+habitat:
+  task:
+    reward_measure: social_nav_reward
+    slack_reward: 0.
+    success_reward: 0.
+    task_spec: multi_agent_social_nav_solved
+    actions:
+      agent_0_arm_action:
+        grip_controller: MagicGraspAction
+      agent_0_oracle_nav_action:
+        agent_index : 0
+        spawn_max_dist_to_obj: -1.0
+      agent_1_oracle_nav_action:
+        motion_control: human_joints
+        spawn_max_dist_to_obj: 1.7
```

Comment:
I think this should be -1. I know it was set like this, but good to change it

Reply:
thank you!